libsolidity/formal/SMTEncoder.h

```diff
@@ -113,9 +113,6 @@ class SMTEncoder: public ASTConstVisitor
 	/// @returns the ModifierDefinition of a ModifierInvocation if possible, or nullptr.
 	static ModifierDefinition const* resolveModifierInvocation(ModifierInvocation const& _invocation, ContractDefinition const* _contract);
 
-	/// @returns the SourceUnit that contains _scopable.
-	static SourceUnit const* sourceUnitContaining(Scopable const& _scopable);
-
```

Comment:
I take it you just happened to notice this is unused and this is in no relation to the fix? Or am I blind at this hour :-)?

Reply:
The bug here was a wrong assumption on `scope()`, so I looked for other uses of `scope()` and found this function, then checked where it's used and it's not so removed it.